celery-entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+set -e
+
+_is_celery_command () {
+  local cmd="$1"; shift
+
+  python - <<EOF
+import sys
+from celery.bin.celery import CeleryCommand
+sys.exit(0 if '$cmd' in CeleryCommand.commands else 1)
+EOF
```

Comment:
The list of commands is long and varies between Celery 3.1 and 4.0 unfortunately 😿